tools/sdk-generation-pipeline/pipelines/steps/parse-and-publish-result.yml

```diff
@@ -88,7 +90,7 @@ steps:
             -u "$(REPO_OWNER)":"$(azuresdk-github-pat)" \
             -H "Accept: application/vnd.github.v3+json" \
             -X POST \
-            -d '{"title":"[AutoPR-${{ parameters.service }}]autogenerated","body":"","base":"'$(sdkMainBranch)'", "head":"$(SDK_PUSH_OWNER):'${branchName}'"}' \
+            -d '{"title":"[AutoPR-${{ parameters.service }}]${{ parameters.swagger.title }}","body":"Create to sync ${{ parameters.swagger.path }}/pull/${{ parameters.swagger.pullRequest }}","base":"'$(sdkMainBranch)'", "head":"$(SDK_PUSH_OWNER):'${branchName}'"}' \
```

Comment:
Slightly orthogonal to this change but have you considered using common create pull request temmplate and script? https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/pipelines/templates/steps/create-pull-request.yml

Reply:
Thx for the suggestion, I read the templates and other scripts which use the templates. Compared with my scenario there are three point block the replace. 
  First: the branch name is calculated from previous step, can we use a dynamic variable in parameters?
  Second: I need to parse the reply of the pulls api to get the pr's number and link. I didn't find any examples to get the api result in another step.
  Third: we will support create multiple PRs in one pipeline, until I get the result from sdk generation docker then I know how may PRs should to create. So I put create pr in a for loop.
Consider these issues, I use my own script.

Reply:
1) Yes you can set the branch via a variable.
2) https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/Submit-PullRequest.ps1#L114, we set a variable with the PR number for other steps that need it.
3) We definitely do have places that call this multiple times. See https://github.com/Azure/azure-sdk-tools/blob/main/eng/pipelines/eng-common-sync.yml#L116 as an example. You could even just call the script instead of using the template if that makes consumption easier.

The main point is that we want to try and avoid duplication as we have a bunch of common scripts/templates that do these common tasks already and we should try to use those instead of re-implementing.

Reply:
Looks good! Thanks for the examples. I will try if I can replace it.